src/plugins/pagination/pager.js

```diff
@@ -11,73 +11,21 @@ TemplatePath
 		};
 	});
 
-const Plugin = PluginComponent('pager', {models: ['pagination']});
+const Plugin = PluginComponent('pager');
+
 class Pager extends Plugin {
 	constructor() {
 		super(...arguments);
-
-		const ctrl = this;
-		this.next = new Command({
-			source: 'pager',
-			execute: () => ctrl.current = ctrl.current + 1,
-			canExecute: () => (ctrl.current + 1) * ctrl.size < ctrl.total
-		});
-
-		this.prev = new Command({
-			source: 'pager',
-			execute: () => ctrl.current = ctrl.current - 1,
-			canExecute: () => ctrl.current > 0
-		});
-	}
-
-	get resource() {
-		return this.model.pagination().resource;
-	}
-
-	get size() {
-		return this.model.pagination().size;
-	}
-
-	set size(value) {
-		this.model.pagination({size: value, current: 0});
-	}
-
-	get sizeList() {
-		return this.model.pagination().sizeList;
-	}
-
-	get current() {
-		return this.model.pagination().current;
-	}
-
-	set current(value) {
-		return this.model.pagination({current: value});
-	}
-
-	get from() {
-		return Math.min(this.total, this.current * this.size + 1);
-	}
-
-	get to() {
-		return Math.min(this.total, (this.current + 1) * this.size);
-	}
-
-	get total() {
-		return this.model.pagination().count;
-	}
-
-	get totalPages() {
-		return Math.max(1, Math.ceil(this.total / this.size));
 	}
 
-	get scroll() {
-		return this.model.scroll();
+	onInit() {
+		this.$scope.$pagerPlugin = new PagerView(this.model);
```

Comment:
rename $pagerPlugin to $pager and revert plugin.pager.html :)

Reply:
Oh, I see. I mixed up `controllerAs` and this one